katcp/server.py

```diff
@@ -2189,19 +2189,28 @@ def request_sensor_value(self, req, msg):
         return req.make_reply("ok", str(len(sensors)))
 
     def request_sensor_sampling(self, req, msg):
-        """Configure or query the way a sensor is sampled.
+        r"""Configure or query the way a sensor is sampled.
```

Comment:
Typo?

Reply:
I think the r is for the code to treat that sentence as raw string

Reply:
Yes, it keeps the linter happy, because we have escaped strings, like `\_` within the docstring.